benchmarks/Microsoft.AspNetCore.Routing.Performance/RoutingBenchmark.cs

```diff
@@ -1,113 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System;
-using System.Linq;
-using System.Threading.Tasks;
-using BenchmarkDotNet.Attributes;
-using Microsoft.AspNetCore.Http;
-using Microsoft.AspNetCore.Routing.Internal;
-using Microsoft.AspNetCore.Routing.Template;
-using Microsoft.AspNetCore.Routing.Tree;
-using Microsoft.Extensions.Logging.Abstractions;
-using Microsoft.Extensions.ObjectPool;
-using Microsoft.Extensions.Options;
-
-namespace Microsoft.AspNetCore.Routing.Performance
-{
-    public class RoutingBenchmark
```

Comment:
I don't think there's any value in keeping this. The new tests are much more sophisticated